### ISSUE TITLE
create a view mode for browsing projects (both public and private)

### ISSUE THREAD

**Issue by williamstein** [sagemathinc/smc#71]
There is a whole group of intended-to-be-implemented functionality that we haven't implemented yet, which would solve many problems.  For now, the best workaround in this case is to share the raw link with others (the link you get when you click "Download...").

Our idea for solving all these problems is to have 3 ways of getting at files, instead of two.  Right now there's:

   - the SMC file browser/editor
   - the raw download server

The third way would be a "view", so the URL would be (say)

   https://cloud.sagemath.com/projects/45878c44..eb0/view/grim_application_paper/TorusN256HD.mp4

but this doesn't exist yet.   Basically with "view" everything would be oriented toward browser, rather than editing.   

This is also needed, e.g., to nicely host a wiki that you edit on SMC and it's the same sort of problem that wiki's have.  SMC is like a wiki with only the edit mode.

**Comment by williamstein** [comment]
Some related tricks: https://cloud.sagemath.com/projects/4a5f0542-5873-4eed-a85c-a18c706e8bcd/files/support/markdown-hints/

**Comment by JCBustamante** [comment]
I want to use SMC to teach. Among other things, I would like to be able to write some .md files, then share the result with my students, via a link in the Moodle page of my course.

When I do exactly this, the students would see the .md file, but what I really want to be shown is the .html obtained from it. So I went to a terminal and made

pandoc -o Ecrire.html EcrireMaths.md

Thus I have the .hmtl, and I can make it public, and share the link. This works with chromium, but not with firefox. For some reason what I got is kind of a blank page, an empty rectangle into which chromium shows the text, but firefox lets it empty. Further, the math characters are not displyed as nicely as they should from this .html file.

The other think I tried is to put all my .md file into a single cell in a .sagews file, with %md, then share it (it is not that big). It works reasonably well, but there is a problem : whenever I enlarge the characters (using te buttons at the top), the sizes of the headers won't vary consistently. Thus, even if the original document looks perfect, enlarging it (for instance during a presentation) will cause that the text structure is somehow lost.
